lib/api/CAnnotationJsonWriter.cc

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+#include <api/CAnnotationJsonWriter.h>
+#include <core/CLogger.h>
+#include <core/CTimeUtils.h>
+
+namespace ml {
+namespace api {
+namespace {
+// JSON field names
+const std::string ANNOTATION_RESULT_TYPE("annotation");
```

Comment:
nit: can we stick with braces. 
```suggestion
const std::string ANNOTATION_RESULT_TYPE{"annotation"};
```
(I've already had case where this has caught bugs for me by flagging up unexpected type conversion, so I think it is worthwhile.)